### PR TITLE
SDK-4039 - Updates ios and android sdk to 7.0.1

### PR DESCRIPTION
**Pull request by akashvercetti**
- Updates ios sdk to 7.0.1
- Updates android sdk to 7.0.1

### CODE REVIEW

File: lib/clevertap_plugin.dart [hunk 1]
```diff
@@ -61,7 +61,7 @@ class CleverTapPlugin {
   static const libName = 'Flutter';
 
   static const libVersion =
-      20401; // If the current version is X.X.X then pass as X0X0X
+      20501; // If the current version is X.X.X then pass as X0X0X
```
Comment (Anush-Shand) [inline lib/clevertap_plugin.dart:64]: Typo here
Should be 20500

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,20 @@
 ## CHANGE LOG
 
+### Version 2.5.0 *(8th October 2024)*
+-------------------------------------------
+**What's new**
+* **[iOS Platform]**
+  * Supports [CleverTap iOS SDK v7.0.1](https://github.com/CleverTap/clevertap-ios-sdk/releases/tag/7.0.1).
+  * Adds support for Custom Code Templates. Please refer to the Custom Code Templates doc to read more on how to integrate this in your app.
```
Comment (Anush-Shand) [inline CHANGELOG.md:8]: Are we supporting this feature in this release?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,20 @@
...
+  * Adds support for File Type Variables in Remote Config. Please refer to the Remote Config Variables doc to read more on how to integrate this in your app.
```
Comment (Anush-Shand) [inline CHANGELOG.md:9]: Are we supporting this feature in this release?

File: doc/Integrate-Android.md [hunk 4]
```diff
@@ -1,111 +1,123 @@
...
-
-In your app's Android Application class add the following code.
-
-```java
-    public class MyApplication extends FlutterApplication {
-        @java.lang.Override
-        public void onCreate() {
-            ActivityLifecycleCallback.register(this); //<--- Add this before super.onCreate()
-            super.onCreate();
+- Add the following to your `dependencies` section in `project/build.gradle`
```
Comment (Anush-Shand) [inline doc/Integrate-Android.md:3]: Note: Majorly only Formatting has been changed in these docs

**Review by Anush-Shand — APPROVED** [review]
LGTM

**Review by darshanclevertap — APPROVED** [review]

**Pull request merged**